src/PlayerManager.cpp

```diff
@@ -184,7 +185,7 @@ void PlayerManager::enterPlayer(CNSocket* sock, CNPacketData* data) {
     std::cout << "\tPC_UID: " << plr.PCStyle.iPC_UID << std::endl;
     )
 
-    response.iID = rand();
+        response.iID = rand();
```

Comment:
You seem to have indented some lines further for no reason?

Reply:
I tend to do CTRL+K then CTRL+D out of habit.